### ISSUE TITLE
Problem with MarketplaceWebService

### ISSUE THREAD

**Issue by cossou** [brickfox/mws#16]
Hi there,

first of all thanks for this library.

I run into a problem when listing order items (`ListOrderItems`).

My problem only affects one special order with Gift Message Text (`<GiftMessageText />` tag on XML).
I believe the text inside this tag contains a `\r\n\r\n` which in this case breaks the logic. 

Check line 531 on the Client.php: 
https://github.com/brickfox/mws/blob/master/src/MarketplaceWebServiceOrders/Client.php#L531

XML (redacted):

```xml 
<?xml version="1.0"?>
<ListOrderItemsResponse xmlns="https://mws.amazonservices.com/Orders/2013-09-01">
  <ListOrderItemsResult>
    <OrderItems>
      <OrderItem>
        <QuantityOrdered>1</QuantityOrdered>
        <Title>XXXX</Title>
        <ShippingTax>
          <CurrencyCode>EUR</CurrencyCode>
          <Amount>0.00</Amount>
        </ShippingTax>
        <PromotionDiscount>
          <CurrencyCode>EUR</CurrencyCode>
          <Amount>0.00</Amount>
        </PromotionDiscount>
        <GiftMessageText>First line --> \r\n\r\n <-- Second line</GiftMessageText>
        <ASIN>xxxx</ASIN>
        <SellerSKU>xxxx</SellerSKU>
        <OrderItemId>xxx</OrderItemId>
        <QuantityShipped>1</QuantityShipped>
        <ShippingPrice>
          <CurrencyCode>EUR</CurrencyCode>
          <Amount>2.99</Amount>
        </ShippingPrice>
        <ItemPrice>
          <CurrencyCode>EUR</CurrencyCode>
          <Amount>44.90</Amount>
        </ItemPrice>
        <ItemTax>
          <CurrencyCode>EUR</CurrencyCode>
          <Amount>0.00</Amount>
        </ItemTax>
        <ShippingDiscount>
          <CurrencyCode>EUR</CurrencyCode>
          <Amount>0.00</Amount>
        </ShippingDiscount>
      </OrderItem>
    </OrderItems>
    <AmazonOrderId>xxxx</AmazonOrderId>
  </ListOrderItemsResult>
  <ResponseMetadata>
    <RequestId>xxxx</RequestId>
  </ResponseMetadata>
</ListOrderItemsResponse>
```

**Comment by rkrx** [comment]
I had this problem too once in a while. I am pretty sure, I applied a hot-fix since I had not much time to test the fix extensively. So instead of having ...
`$responseComponents = preg_split("/(?:\r?\n){2}/", $response);`
... I changed the line to ...
`$responseComponents = preg_split("/(?:\r?\n){2}/", $response, 2);`
... iirc.

Would you make a PR with the changes? If so, could you fix all the `_extractHeadersAndBody`-Functions? Some already have this limit parameter set.

**Comment by cossou** [comment]
Thanks for the fast reply @rkrx
Such an easy fix! Tomorrow morning I will send you a PR.

**Comment by nikolashermann** [comment]
Thanks to both of you, the PR looks good to me and is merged in v0.0.10

**Comment by cossou** [comment]
Thanks @nikolashermann you can also close pr #15 and #13
